hack/run-tests.sh

```diff
@@ -5,10 +5,10 @@ set -e
 source hack/common.sh
 
 prefix=${DOCKER_PREFIX:-kubevirt}
-tag=${DOCKER_TAG:-v0.12.0-alpha.2}
+tag=${DOCKER_TAG:-v0.13.0}
 kubeconfig=${KUBECONFIG:-~/.kube/config}
 [ -z "$OC_PATH" ] && OC_PATH=$(command -v oc)
 [ -z "$KUBECTL_PATH" ] && KUBECTL_PATH=$(which kubectl)
 [ -z "$VIRTCTL_PATH" ] && VIRTCTL_PATH=$(which virtctl)
 
-${TESTS_OUT_DIR}/tests.test -kubeconfig=$kubeconfig -tag=$tag -prefix=$prefix -oc-path=${OC_PATH} -kubectl-path=${KUBECTL_PATH} -virtctl-path=${VIRTCTL_PATH} ${FUNC_TEST_ARGS}
+${TESTS_OUT_DIR}/tests.test -kubeconfig=$kubeconfig -container-tag=$tag -container-prefix=$prefix -oc-path=${OC_PATH} -kubectl-path=${KUBECTL_PATH} -virtctl-path=${VIRTCTL_PATH} ${FUNC_TEST_ARGS}
```

Comment:
@lukas-bednar should it be "-container-tag" or "-docker-tag" ?

Reply:
I had checked with @lukas-bednar yesterday and it was he who suggested that it is now s/tag/container-tag/, only after updating this, that i was able to move ahead and hit the 3 test-case failures that we are seeing now. 
So, I can confirm this.